docs/workforce-management-framework/topics/con-KeycloakVsPassport.adoc

```diff
@@ -0,0 +1,37 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} currently supports two security implementations
+Keycloak and Passport.js.
+
+== Passport.js
+Passport.js is a lightweight authentication middleware for Node.js that is Extremely flexible and modular.
+A comprehensive set of strategies support authentication using a username and password, Facebook, Twitter, and more.
+For more information see link:https://passport.org[passport.org].
+
+== Keycloak
+Keycloak is a full identity management solution with many features
+
+- link:https://keycloak.gitbooks.io/documentation/server_admin/topics/identity-broker.html[Identity brokering]
+- link:https://keycloak.gitbooks.io/documentation/server_admin/topics/authentication/x509.html[Client cert auth]
+- link:https://keycloak.gitbooks.io/documentation/server_admin/topics/authentication/otp-policies.html[OTP 2FA]
+- link:https://keycloak.gitbooks.io/documentation/server_admin/topics/sessions/administering.html[User session management]
+- link:https://keycloak.gitbooks.io/documentation/server_admin/topics/events.html[Auditing and Events]
+- link:https://keycloak.gitbooks.io/documentation/server_admin/topics/authentication/password-policies.html[Password Policy Manager]
+
+For a full feature list, see link:https://keycloak.gitbooks.io/documentation/server_admin/topics/overview/features.html[Keycloak]
+
+
+== Which to use?
+
+Both solutions have there benefits
+
+*Keycloak*
+
+If you are looking for a full identity management solution that can be tied into
+existing *saml/ldap/sso* solutions then Keycloak should be your choice. Keycloak when running is an separate
+identity management server with all the overheads involved in running a server.
+
+*Passport.js*
+
+If you are looking for a standalone authentication solution that is ready out of the box then passport.js option it for you.
```

Comment:
passport.js option is for you sounds strange. 
Let's just propose facts. maybe points?

``Passport```
- node.js based
- No extra server required
- Available by default when running server application
- Covers authentication and authorization only

``Keycloak``
- ...

etc.